packages/migration-graph/src/migration-strategy.ts

```diff
@@ -78,7 +76,7 @@ export function getMigrationStrategy(
   rootDir = fs.realpathSync(rootDir);
   const m = buildMigrationGraph(rootDir, options);
 
-  const strategy = new MigrationStrategy(m.sourceType);
+  const strategy = new MigrationStrategy(m.sourceType as SourceType);
```

Comment:
if we can avoid type casting here we should. `m.sourceType` should know its type

Reply:
This is a hack around a circular dependency between `migration-graph` package and the implementations in `migration-graph-shared` and `migration-graph-ember`. 

The type for SourceType would need to exists in all three packages. Either we need yet another package, or we hand roll a `d.ts` to get around the circular (Unsure how to do this right now).

I will add a TODO to drop or refactor this.